CONTRIBUTING.md

```diff
@@ -52,18 +52,18 @@ and 2017 would be `swb22017`).
 (`Day of Defeat` -> `dod`), then the new name should ignore rule #2 
 (`Day of Dragons` -> `dayofdragons`).
 5. Roman numbering will be converted to arabic numbering (`XIV` -> `14`).
-6. Unless numbers are at the end of a name, they will be considered words, 
-but digits will always be used instead of the acronym (counter to #2) (`Left 4 
-Dead` -> `l4d`) unless they at the start position (`7 Days to Die` -> `sdtd`), 
-if they are at the end (such as sequel number or the year), always append them 
-(`Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` -> 
-`unrealtournament2003`).
+6. Unless numbers (years included) are at the end of a name, they will be considered 
+words. Digits will be used instead of the acronym of that number's digits (`Left 4 
+Dead` -> `l4d`) unless they are at the start position (`7 Days to Die` -> `sdtd`). 
+Other examples: `Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` ->
+`unrealtournament2003`.
 7. If a game supports multiple protocols, multiple entries will be done for 
 said game where the edition/protocol name (first disposable in this order) will 
-be appended to the game name (Minecraft is divided by 2 editions, Java and Bedrock 
-which will be `minecraftjava` and `minecraftbedrock` respectively) and one more 
-entry can be added by the base name of the game which queries in a group said 
-supported protocols to make generic queries easier and disposable.
+be appended to the base game id's fully, ignoring #2 (Minecraft is mainly divided by 
+2 editions, Java and Bedrock which will be `minecraftjava` and `minecraftbedrock` 
+respectively) and one more entry can be added by the base name of the game which 
+queries in a group said supported protocols to make generic queries easier and 
+disposable.
```

Comment:
Here we're still missing how specifically protocols should be encoded:
- Should we append the protocol exactly as it is in the brackets
- Should we follow the same rules for encoding an game name then append that
- Should we append the words from the protocol to the game name and then apply the other rules

Reply:
I thought it was clear, re-reading it again for the 7th time I realised it was not, updated.